CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing to the **RDA FAIR Mapping Working Group**
+
+We welcome **all** contributions — from typo fixes to new case studies.  
+This short guide explains the preferred workflow for each contribution type.  
+If something is unclear, open a [GitHub Discussion](../../discussions) and we’ll help.
```

Comment:
What exactly is the difference between issues and discussion in the context of this repo? What if somebody opens an issue for an unclear issue? Some kind of summary of issues vs. discussion would be helpful.